odl/discr/lp_discr.py

```diff
@@ -144,11 +166,15 @@ def interp(self):
         """Interpolation type of this discretization."""
         return self._interp
 
-    def points(self):
-        return self.grid.points(order=self.order)
+    def _inner(self, x1, x2):
+        """Raw inner product of two vectors."""
+        if self.exponent != 2.0:
```

Comment:
I dont think we should have these tests, easier to simply pass them to `super()` since the `Rn` space should complain later anyway.